include/stun.hrl

```diff
@@ -64,34 +64,19 @@
 -define(STUN_ATTR_PRIORITY, 16#0024).
 -define(STUN_ATTR_USE_CANDIDATE, 16#0025).
 -define(STUN_ATTR_FINGERPRINT, 16#8028).
--define(STUN_ATTR_ICE_CONTROLLED, 16#0029).
--define(STUN_ATTR_ICE_CONTROLLING, 16#002a).
+-define(STUN_ATTR_ICE_CONTROLLED, 16#8029).
+-define(STUN_ATTR_ICE_CONTROLLING, 16#802a).
 
 -record(stun,
         {class = request :: request | response | error | indication,
          method = ?STUN_METHOD_BINDING :: non_neg_integer(),
-         magic = ?STUN_MAGIC :: non_neg_integer(),
-         trid = 0 :: non_neg_integer(),
-         raw = <<>> :: binary(),
-         unsupported = [],
-         'ALTERNATE-SERVER',
-         'CHANNEL-NUMBER',
-         'DATA',
-         'DONT-FRAGMENT' = false,
-         'ERROR-CODE',
-         'LIFETIME',
-         'MAPPED-ADDRESS',
-         'MESSAGE-INTEGRITY',
-         'NONCE',
-         'REALM',
-         'REQUESTED-ADDRESS-FAMILY',
-         'REQUESTED-TRANSPORT',
-         'SOFTWARE',
-         'UNKNOWN-ATTRIBUTES' = [],
-         'USERNAME',
-         'XOR-MAPPED-ADDRESS',
-         'XOR-PEER-ADDRESS' = [],
-         'XOR-RELAYED-ADDRESS'}).
+         magic = ?STUN_MAGIC :: non_neg_integer(), trid = 0 :: non_neg_integer(),
```

Comment:
I don't think it looks better now. Either don't format this file or set formatter in such a way it doesn't mess this up. Please check other places too

Reply:
I have stopped formatting this file, but there are still some other places with record definitions, that are formatted in this messy way. I have also checked alternate Erlang formatters, but this old one, used currently by us, seems to be the best, despite the fact, that is not formatting records in the prettiest way